ctapipe_io_magic/__init__.py

```diff
@@ -17,6 +17,7 @@
 from astropy.time import Time
 from pkg_resources import resource_filename
 
+from ctapipe_io_lst.constants import REFERENCE_LOCATION
```

Comment:
mmmh I would not have introduced a dependency also on ctapipe_io_lst ... I would assume that those coordinates will not change frequently (if at all, maybe when more than one LST is included, but that's quite in the future). I would probably go into the direction of including those constants in constants.py and update them if really needed. @jsitarek opinions?